### PR TITLE
[WIP] New Report Page

### PR DESCRIPTION
**Pull request by HarshKhandeparkar**
## PR Checklist

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes
- New, improved report pages in `/$/reportContent` and `/$/reportDmca`
- DMCA reports and Normal reports are separate

Issue Number:

### CODE REVIEW

**Comment by HarshKhandeparkar** [comment]
@seanyesmunt I need your help again. Any POST requests should be made in the source code of lbry-desktop or should it be a new action in lbry-redux?

**Comment by Eniamza** [comment]
@HarshKhandeparkar Edits are clear.. So I don't think any markup editing is needed

File: ui/modal/modalConfirmReport/view.jsx [hunk 1]
```diff
@@ -23,10 +23,16 @@ const ModalConfirmReport = (props: Props) => {
...
-        FORMS.
+        <b>DISCLAIMER</b>:{' '}
       </p>
+      <br />
+      <hr />
+      <p>
+        {' '}
+        A minimal amount of LBC will be sent to LBRY Inc as a deposit. If your report is valid, You will get a{' '}
+        <b>FULL RETURN</b>. This validation is necessary for preventing Bots and Report Abuse.
+      </p>
```
Comment (TigerxWood) [inline ui/modal/modalConfirmReport/view.jsx]: Can you please make this translatable? Like: {__('A minimal amount of LBC will be sent to LBRY Inc as a deposit. If your report is valid, You will get a <b>FULL RETURN</b>. This validation is necessary for preventing Bots and Report Abuse.')}

File: ui/modal/modalDmca/view.jsx [hunk 2]
```diff
@@ -62,6 +66,10 @@ const ModalDmca = (props: Props) => {
       contentLabel="Copyright Infringement Report"
```
Comment (TigerxWood) [inline ui/modal/modalDmca/view.jsx]: Maybe also this: {__('Copyright Infringement Report')} to be translatable?

File: ui/modal/modalDmca/view.jsx [hunk 3]
```diff
@@ -62,6 +66,10 @@ const ModalDmca = (props: Props) => {
...
       title={__('Copyright Infringement Report')}
     >
+      <blockquote>
+        <b>claimId</b>: {claimId}
```
Comment (TigerxWood) [inline ui/modal/modalDmca/view.jsx]: Can you find a way to make <b>claimId</b> translatable?

**Comment by HarshKhandeparkar** [comment]
As you wish...

**Comment by HarshKhandeparkar** [comment]
> @seanyesmunt I need your help again. Any POST requests should be made in the source code of lbry-desktop or should it be a new action in lbry-redux?
@seanyesmunt ?

**Comment by HarshKhandeparkar** [comment]
Ok ty.

**Comment by Eniamza** [comment]
@HarshKhandeparkar How many points did we add?

**Comment by HarshKhandeparkar** [comment]
points=? lol

**Comment by seanyesmunt** [comment]
@HarshKhandeparkar @Eniamza Is this still being worked on?

**Comment by HarshKhandeparkar** [comment]
We need help with the server side :( and haven't found anyone.

**Comment by Eniamza** [comment]
@seanyesmunt Before you forget, again we need someone to add a few line of codes to make a post request

**Comment by HarshKhandeparkar** [comment]
> @seanyesmunt Before you forget, again we need someone to add a few line of codes to make a post request

I can do that btw, I just dk where to send the request and what exactly to send because there is no one to receive.

**Comment by seanyesmunt** [comment]
Sorry for the delay. I've been caught up getting a new app release out. You can make the post request to lbry.com, the same way we are doing it currently. I think you will need to update this file https://github.com/lbryio/lbry.com/blob/master/controller/action/ReportActions.class.php

@kauffj can you confirm that is the file that will need to be updated so that the new report page can have additional flags?

**Comment by kauffj** [comment]
@seanyesmunt I'd kind of rather bite the bullet on moving this to IAPIs, we should consider lbry.com deprecated for feature purposes.

@tiger5226 do you have time to prioritize moving a simple API endpoint for DMCA reporting to IAPIs? It's currently completely stateless -- it just sends an email.

Specifically, it's a POST that contains these 4 values and executes this relatively simple logic:
https://github.com/lbryio/lbry.com/blob/master/controller/action/ReportActions.class.php#L64

**Comment by kauffj** [comment]
Apologies @HarshKhandeparkar but I'm probably going to hold this up a bit longer :cry:

**Comment by HarshKhandeparkar** [comment]
Take your sweet time. We have got used to this ;)

**Comment by Eniamza** [comment]
@HarshKhandeparkar Agree

**Comment by seanyesmunt** [comment]
@Eniamza @HarshKhandeparkar We are working on creating a new endpoint for dmca reports. Then we will create additional endpoints for the other types of reports you have added. I will let you know when that is merged!

**Comment by seanyesmunt** [comment]
@HarshKhandeparkar Sorry this has been sitting for so long. I'm going to close this since we are pretty swamped right now and won't get to the API changes for a while. Please keep these changes in your fork so we can re-open and get this merged in once we get some time and the APIs are created.

**Pull request closed without merging**